Sources/StreamChatUI/ChatMessageList/RETHINK/MessageListVC.swift

```diff
@@ -167,80 +169,6 @@ class MessageListVC<ExtraData: ExtraDataTypes>: _ViewController, UICollectionVie
         return delay > minTimeIntervalBetweenMessagesInGroup
     }
     
-    func cellLayoutOptionsForMessage(at indexPath: IndexPath) -> ChatMessageLayoutOptions {
```

Comment:
As mentioned in Slack "it's important to keep the cellLayoutOptionsForMessage method so users can easily override it"

Reply:
✅